src/jit/gentree.cpp

```diff
@@ -7621,6 +7621,39 @@ GenTree* Compiler::gtNewBlkOpNode(
     return result;
 }
 
+//------------------------------------------------------------------------
+// gtNewPutArgReg: Creates a new Obj node.
```

Comment:
This should say "Creates a new PutArgReg node." (or "GT_PUTARG_REG" node, or something like that).